MarsPizzaParty.md

```diff
@@ -0,0 +1,28 @@
+Title: Mars Pizza Party
```

Comment:
Can you make the title of your file match the way that other event posts are already named?